src/Qowaiv.Validation.Abstractions/Result_TModel.cs

```diff
@@ -85,7 +85,7 @@ public Result<TOut> Act<TOut>(Func<TModel, Result<TOut>> action)
             return new(outcome.IsValid
                 ? outcome.Value
                 : default,
-                ((FixedMessages)Messages).AddRange(outcome.Messages));
+                ((FixedMessages)Messages).AddRange(outcome.Messages).Add(ActFailed.New(outcome.IsValid)));
```

Comment:
I think adding a new message might not be visible enough, especially if you have many other error messages. I would prefer to see something more in line with debugger breakpoints.